packages/modules/src/modules/epics/ContributionsEpicCtas.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { createViewEventFromTracking, createInsertEventFromTracking } from '@sdc/shared/lib';
+import { EpicProps } from '@sdc/shared/types';
+import { ContributionsEpicReminder } from './ContributionsEpicReminder';
+import { ContributionsEpicButtons } from './ContributionsEpicButtons';
+import { defineFetchEmail } from '../shared/helpers/definedFetchEmail';
+
+interface OnReminderOpen {
+    buttonCopyAsString: string;
+}
+
+export const ContributionsEpicCtas: React.FC<EpicProps> = ({
+    variant,
+    countryCode,
+    articleCounts,
+    tracking,
+    submitComponentEvent,
+    onReminderOpen,
+    email,
+    fetchEmail,
+}: EpicProps): JSX.Element | null => {
+    useEffect(() => {
+        if (submitComponentEvent) {
```

Comment:
I don't think we should be using `submitComponentEvent` here - these are the INSERT/VIEW events for the epic as a whole

Reply:
Removed duplicate code